pulsar-client-auth-athenz/src/main/java/org/apache/pulsar/client/impl/auth/AuthenticationAthenz.java

```diff
@@ -172,15 +174,20 @@ private PrivateKey loadPrivateKey(String privateKeyURL) {
                 privateKey = Crypto.loadPrivateKey(new File(privateKeyURL));
             } else if (uri.getScheme().equals("file")) {
                 privateKey = Crypto.loadPrivateKey(new File(uri.getPath()));
-            } else if(uri.getScheme().equals("data")) {
+            } else if (uri.getScheme().equals("data")) {
                 List<String> dataParts = Splitter.on(",").splitToList(uri.getSchemeSpecificPart());
-                if (dataParts.get(0).equals("application/x-pem-file;base64")) {
+                // Support Urlencode but not decode here because already decoded by URI class.
+                if (dataParts.get(0).equals("application/x-pem-file")) {
```

Comment:
It would be better to use constants for the String values

Reply:
Fixed.